### PR TITLE
Add Emergency Broadband Benefit

### PR DESCRIPTION
**Pull request by MaxGhenis**
# New variable

- [x] Label field added
- [x] Documentation field added
- [x] Unit field added
- [x] Variable name follows conventions
- [x] Unit test(s) added
- [x] Integration test(s) added if relevant

### CODE REVIEW

File: openfisca_us/parameters/fcc/ebb/categorical_eligibility.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+description: Program eligibility or participation that entitle a household to the Emergency Broadband Benefit (EBB) program
```
Comment (nikhilwoodruff) [inline openfisca_us/parameters/fcc/ebb/categorical_eligibility.yaml:1]:
```suggestion
description: Program eligibility or participation that entitles a household to the Emergency Broadband Benefit (EBB) program
```

**Review by nikhilwoodruff — APPROVED** [review]

File: openfisca_us/parameters/fcc/ebb/prior_enrollment_required.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+description: Whether a household must already be enrolled in the Emergency Broadband Benefit to receive it
+values:
+  2021-05-12: false
+  2022-01-01: true
+metadata:
+  reference:
+    - name: WCB Issues Guidance for Transition to Affordable Connectivity Program
```
Comment (nikhilwoodruff) [inline openfisca_us/parameters/fcc/ebb/prior_enrollment_required.yaml:7]:
```suggestion
    - title: WCB Issues Guidance for Transition to Affordable Connectivity Program
```

**Review by nikhilwoodruff — APPROVED** [review]
Minor comment on the reference, nothing breaking, LGTM after that though, thanks

**Comment by MaxGhenis** [comment]
I blame Copilot! Thanks.

**Pull request merged**